### PR TITLE
Add views

### PR DESCRIPTION
**Pull request by rbd3**
# Change made :
👉 Add user views
👉 Add post views

### CODE REVIEW

File: app/views/posts/_post.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="post-container">
+  <span class="post-title"><%= post.title %></span>
+  <span class="post-bio"><%= post.text %></span>
+  <span class="post-totalpost">Comments : <%= post.comments_counter %>, Likes : <%= post.likes_counter %></span>
+</div>
```
Comment (OyePriscilla) [inline app/views/posts/_post.html.erb]: - [ ] Please for good user experience; kindly make the individual post to be a clickable element that can lead us to its `show_page`. I was only able to access the `post_show` page via the url. 
- [ ] Kindly ensure that it is designed in accordance to the wirefreme ( the number of likes is not properly placed).

Untitled

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -13,3 +13,149 @@
...
+  white-space: nowrap;
+}
+
+.recent-comments {
+  margin-left: 2%;
+}
+
+.all-comments {
+  margin-left: 2%;
+}
```
Comment (OyePriscilla) [inline app/assets/stylesheets/application.css:161]: - [ ] Please kindly avoid code repetition in order for your codebase to be neat. ℹ️

Comment (OyePriscilla) [inline app/assets/stylesheets/application.css]: - [ ] Please kindly avoid code repetition in order for your codebase to be neat. ℹ️

File: app/views/posts/show.html.erb [hunk 3]
```diff
@@ -1,2 +1,24 @@
...
+    <h3 class="comment-title">
+      Post # <%= @index %>
+      by <%= @user.name %>
+    </h3>
 
+    <div class="counter-wrapper">
+      <p class="comment-count">Comments: <%= @post.comments.count %></p>
+      <p class="like-count">Likes: <%= @post.likes.count %></p>
+    </div>
+
```
Comment (OyePriscilla) [inline app/views/posts/show.html.erb:11]: - [ ] Kindly ensure that it is designed in accordance to the wirefreme ( the number of likes is not properly placed).

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @rbd3 👏 👏 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Use preprocessed HTML file with embedded Ruby code. ✔️
Use layouts and templates for shared content. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @rbd3,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ]  Followed Gitflow 💯
- [ ]  Descriptive PR⭐
- [ ]  Kudos for working on the views🤩
- [ ] Great job working on the requested changes 💯


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**